service/customer/internal/db/database.go

```diff
@@ -0,0 +1,49 @@
+package db
+
+import (
+	"fmt"
+
+	"github.com/GenesisEducationKyiv/software-engineering-school-4-0-boredgus/service/customer/internal/repo"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"gorm.io/gorm/schema"
+)
+
+var models = map[string]interface{}{
```

Comment:
it could be repo.Customer instead of interface

Reply:
I considered that this map is redundant and got rid of it at all.